mandatory/4-tax.js

```diff
@@ -17,7 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(money) {
+  money=((money/100)*20+money);
```

Comment:
Here, you should use your previous function. There is this concept call DRY (Don't repeat yourself). If you already wrote a piece code that does the same, then you should use it. For example:

`calculateSalesTax(money)`

Reply:
Thanks for commenting my work , i made changes :)